src/assets/KeyIcon.tsx

```diff
@@ -1,11 +1,22 @@
 import { ReactElement } from 'react'
 
-export default function KeyIcon(): ReactElement {
+interface KeyIconProps {
+  className?: string
+}
+
+export default function KeyIcon({ className }: KeyIconProps): ReactElement {
```

Comment:
Again, you do not really need to add type annotation on components!
```suggestion
export default function KeyIcon({ className }: KeyIconProps) {
```